torchgen/api/autograd.py

```diff
@@ -312,6 +312,10 @@ def match_differentiability_info(
     }
 
     def find_info(f: NativeFunction) -> Tuple[Optional[DifferentiabilityInfo], bool]:
+        # Don't bother matching info to generated out= variants
+        if f.func.kind() == SchemaKind.out:
```

Comment:
Hmm, should this be `if "generated" in f.tags and f.func.kind() == SchemaKind.out`? We probably don't need to bother routing the old out= ops through this logic, just the new generated ones.